amqp-quickstart/amqp-quickstart-processor/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+# Configure the incoming AMQP queue `quote-requests`
+mp.messaging.incoming.requests.connector=smallrye-amqp
+mp.messaging.incoming.requests.address=quote-requests
```

Comment:
In the Kafka one, we don't configure this and call the channel `quote-requests`. Should we stay consistent?

Reply:
Nevermind, wrong service :)